code/game/objects/structures/stool_bed_chair_nest/wheelchair.dm

```diff
@@ -195,4 +195,33 @@
 
 /proc/equip_wheelchair(mob/living/carbon/human/H) //Proc for spawning in a wheelchair if a new character has no legs. Used in new_player.dm
 	var/obj/structure/bed/chair/wheelchair/W = new(H.loc)
-	W.buckle_mob(H)
\ No newline at end of file
+	W.buckle_mob(H)
+
+/obj/item/wheelchair
+	name = "wheelchair"
+	desc = "A folded wheelchair that can be carried around."
+	icon = 'icons/obj/furniture.dmi'
+	icon_state = "wheelchair_folded"
+	item_state = "wheelchair"
```

Comment:
`item_state` is using for "in hand" overlay.
May be it's better to skip it or replace with generic sprite.